spec/instant.html

```diff
@@ -659,17 +659,10 @@ <h1>
         1. Else,
           1. Let _first_ be _other_.
           1. Let _second_ be _instant_.
-        1. Set _options_ to ? GetOptionsObject(_options_).
-        1. Let _smallestUnit_ be ? ToSmallestTemporalUnit(_options_, « *"year"*, *"month"*, *"week"*, *"day"* », *"nanosecond"*).
-        1. Let _defaultLargestUnit_ be ! LargerOfTwoTemporalUnits(*"second"*, _smallestUnit_).
-        1. Let _largestUnit_ be ? ToLargestTemporalUnit(_options_, « *"year"*, *"month"*, *"week"*, *"day"* », *"auto"*, _defaultLargestUnit_).
-        1. Perform ? ValidateTemporalUnitRange(_largestUnit_, _smallestUnit_).
-        1. Let _roundingMode_ be ? ToTemporalRoundingMode(_options_, *"trunc"*).
-        1. Let _maximum_ be ! MaximumTemporalDurationRoundingIncrement(_smallestUnit_).
```

Comment:
Looks like this is adding a `NegateTemporalRoundingMode` call in the `since` case that wasn't there before

Reply:
good point

Reply:
Would you mind arranging the commits so that one commit refactors DifferenceTemporalInstant to use the sign instead of swapping the order, and the second commit adds GetDifferenceOptions?

Reply:
I wish I know how to do that.

Reply:
@ptomato I do not know how to do what you are asking me to do. Can we just squash these commit together and force push instead?

Reply:
Actually it seems the first part was already done in #2241, I will just rebase this and merge it.